docs-src/getting-started.md

```diff
@@ -0,0 +1,56 @@
+# Getting started with Services APIs
+
+## Installing CRDs
+
+This project provides a collection of Custom Resource Definitions (CRDs)that can
+be installed into any Kubernetes (>= 1.16) cluster.
+
+To install the CRDs, please execute:
+
+```
+kubectl kustomize "github.com/kubernetes-sigs/service-apis/config/crd?ref=master" \
+| kubectl apply -f -
+```
+
+## Install an implementation
+
+Multiple projects implement the APIs defined by this project.
+You will need to either install an implementation or verify if one is already
+setup for your cluster.
+
+## Sample Gateway
+
+Once you have the CRDs and an implementation installed, you are ready to
+use Service APIs.
+
+In this example, we are installing three resources:
+
+- An `acme-lb` GatewayClass which is being managed by a `acme.io/gateway-controller`
+  controller running in the cluster.
```

Comment:
Maybe should add a note that users may want to use a GatewayClass provided by the implementation installed in their cluster. If one is not provided, they should ensure that the controller name used here matches a controller name supported by an implementation installed in their cluster.

Reply:
> If one is not provided

Skipped this part. We shouldn't be encouraging implementers not providing a GatewayClass right now.